lib/browser_core.js

```diff
@@ -141,4 +144,21 @@ module.exports = class BrowserCore {
         });
         return Promise.all(promises);
     }
+
+    _stringifyLogText(log) {
```

Comment:
These methods are duplicate in browser console. Could we move the whole logic of processing raw logs to a separate class/util methods? This way we avoid duplicate code and we could use the Log class for pageLog instead of raw logs

Reply:
would you like me to add it to the already existing `utils` file? or create a separate file for this?

Reply:
Both options seem fine to me, although the current utils file is getting a bit large

Reply:
done